frontend/src/app/active-sessions/upload-dialog/upload-dialog.component.html

```diff
@@ -0,0 +1,113 @@
+<!--
+ ~ Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<h1>Upload Files (Experimental)</h1>
+
+<mat-progress-bar
+  class="progress-bar"
+  mode="indeterminate"
+  *ngIf="loadingFiles"
+>
+</mat-progress-bar>
+<div class="progress">
+  <mat-progress-bar
+    class="progress-bar"
+    mode="determinate"
+    [value]="uploadProgress"
+    *ngIf="uploadProgress"
+  >
+  </mat-progress-bar>
+  <mat-icon
+    class="cancel-upload"
+    (click)="cancelUpload()"
+    *ngIf="uploadProgress"
+    >delete_forever</mat-icon
```

Comment:
```suggestion
    >cancel</mat-icon
```

`delete_forever` is probably not a good icon to cancel an upload.